### PR TITLE
Fix typos in the Doc directory

### PR DESCRIPTION
**Pull request by cclauss**
https://pypi.org/project/codespell enabled me to find and fix these typos.

### CODE REVIEW

File: Doc/library/curses.rst [hunk 1]
```diff
@@ -936,7 +936,7 @@ the following methods and attributes:
 
 .. method:: window.getbegyx()
 
-   Return a tuple ``(y, x)`` of co-ordinates of upper-left corner.
+   Return a tuple ``(y, x)`` of coordinates of upper-left corner.
```
Comment (ambv) [inline Doc/library/curses.rst]:
```suggestion
   Return a tuple ``(y, x)`` of co-ordinates of upper-left corner.
```

It's an archaic spelling for sure but it's not a typo. I'd leave this as is.

Comment (cclauss) [reply Doc/library/curses.rst]: I thought the same but then even Google corrected this...  https://www.google.com/search?q=co-ordinates

Committing your suggestion.

File: Doc/library/crypt.rst [hunk 2]
```diff
@@ -111,7 +111,7 @@ The :mod:`crypt` module defines the following functions:
    .. index:: single: crypt(3)
 
    Since a few :manpage:`crypt(3)` extensions allow different values, with
-   different sizes in the *salt*, it is recommended to use  the full crypted
+   different sizes in the *salt*, it is recommended to use the full encrypted
```
Comment (ambv) [inline Doc/library/crypt.rst]: Usage of the term "crypted" is deliberate here. There's a code example below with a variable "cryptedpasswd". It's a rare usage but valid:

https://www.google.com/search?q=%22crypted+password%22

The intended meaning is that a "crypted password" is a password that was passed through `crypt()`. Thus, it is not retrievable again, unlike regularly "encrypted" passwords which can still be "decrypted". Moreover, for this particular example,the  "crypted password" also contains an embedded salt which is crucial in checking whether the password is valid. Consider:

```pycon
>>> cleartext = 'scott-tiger'
>>> salt = crypt.mksalt()
>>> cryptedpasswd = crypt.crypt(cleartext, salt)
>>> is_password_valid = crypt.crypt(cleartext, cryptedpasswd) == cryptedpasswd
>>> is_password_valid
True
>>> is_password_valid = crypt.crypt('tiger-scott', cryptedpasswd) == cryptedpasswd
>>> is_password_valid
False
```

```suggestion
   different sizes in the *salt*, it is recommended to use  the full crypted
```

File: Doc/library/crypt.rst [hunk 3]
```diff
@@ -111,7 +111,7 @@ The :mod:`crypt` module defines the following functions:
    .. index:: single: crypt(3)
 
    Since a few :manpage:`crypt(3)` extensions allow different values, with
-   different sizes in the *salt*, it is recommended to use  the full crypted
+   different sizes in the *salt*, it is recommended to use the full crypted
```
Comment (ambv) [inline Doc/library/crypt.rst]: Sorry, it's not worth touching the line to remove one extra space. Every change makes looking through git history a bit more time consuming so trivial changes that don't impact the reader should be avoided.

```suggestion
   different sizes in the *salt*, it is recommended to use  the full crypted
```

**Comment by ambv** [comment]
Due to the number of conflicting changes in the docs, it's not feasible to generate backports of this PR. @cclauss, if you could create new PRs to the same effect against the 3.10 and 3.9 branches respectively, that'd be great.

**Pull request merged**